src/DIRAC/Resources/Computing/AREXComputingElement.py

```diff
@@ -611,27 +683,119 @@ def getJobOutput(self, jobID, _localDir=None):
         if not stamp:
             return S_ERROR("Pilot stamp not defined for %s" % pilotRef)
 
-        # Prepare the command
+        # Prepare the command: Get output files
         command = "jobs/"
         job = self._DiracToArcID(pilotRef)
-        query = self._UrlJoin([command, job, "session", stamp, ".out"])
+        query = self._UrlJoin([command, job, "diagnose"])
+        query += "errors"
         if query.startswith("Unknown"):
             return S_ERROR("Problem creating REST query %s" % query)
 
         # Submit the GET request to retrieve outputs
+        self.log.debug("Retrieving pilot logging info for %s" % pilotRef)
         response = self.session.get(query, headers=self.headers, timeout=self.arcRESTTimeout)
         if not response.ok:
-            self.log.error("Error downloading stdout", "for %s: %s" % (job, response.text))
-            return S_ERROR("Failed to retrieve at least some output for %s" % jobID)
-        output = response.text
+            self.log.error(
+                "Error downloading logging info", "for %s: %s %s" % (job, response.status_code, response.reason)
+            )
+            return S_ERROR("Failed to retrieve logging info for %s" % jobID)
+        loggingInfo = response.text
+
+        return S_OK(loggingInfo)
 
-        query = self._UrlJoin([command, job, "session", stamp, ".err"])
+    #############################################################################
+
+    def _getListOfAvailableOutputs(self, pilotRef, job):
```

Comment:
`job` is the same as `jobID` (so `pilotID`)?

Reply:
Here `pilotRef` becomes `jobID` and `job` becomes `arcJobID`.